prbt_support/urdf/prbt_macro.xacro

```diff
@@ -187,7 +187,7 @@ limitations under the License.
       <parent link="${prefix}link_3"/>
       <child link="${prefix}link_4"/>
       <axis xyz="0 0 1" />
-      <limit effort="176" velocity="1.57" lower="-2.967" upper="2.967"/>
+      <limit effort="176" velocity="1.57" lower="-2.9671" upper="2.9671"/>
```

Comment:
why 4-precision? Why not more? Why not less?

Reply:
Good question. From what I know, the precision of the axis is up to 1/1000 degree. This corresponds to 1.7e-5 in radians. So at least 5-precision is sufficient. I changed it accordingly.